Fix MYSQL Access Denied Issue on restart

BLasan opened the pull request:
## Description

See [FINERACT-1294
](https://issues.apache.org/jira/browse/FINERACT-1294)

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Write the commit message as per https://github.com/apache/fineract/#pull-requests

- [x] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [x] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

- [ ] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/api-docs/apiLive.htm with details of any API changes

- [x] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)

FYI our guidelines for code reviews are at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

vorburger commented on kubernetes/kubectl-startup.sh:
```diff
@@ -19,7 +19,7 @@
 
 echo "Setting Up Fineract service configuration..."
-kubectl create secret generic fineract-tenants-db-secret --from-literal=username=root --from-literal=password=$(head /dev/urandom | tr -dc A-Za-z0-9 | head -c 16)
+kubectl create secret generic fineract-tenants-db-secret --from-literal=username=root --from-literal=password=root
```
Wait a second... doesn't this hardcode it to"root"?! That's..  probably not such a good idea? 😎

BLasan replied:
Actually these usernames and passwords will be saved as secrets.  But anyway you can get those values by running the command `kubectl get secrets/fineract-tenants-db-secret --template={{.data.password}} | base64 --decode` . And the problem in here is that when we use a random password, after restarting the container we'll be getting access denied issue.

BLasan replied:
I do have another solution for this. Not sure whether it'll help to sort out this issue. Let me try that way as well :)

BLasan replied:
I have used an initcontainer instead of hardcoding the password. So before the container creation it will bind /var/lib/mysql directory which could help us to get rid of this "Access Denied" issue after restarting the container

BLasan replied:
@vorburger Please review the new changes

BLasan commented:
@vorburger  @xurror  This issue will occur if we do not remove volume mounted inside minikube node when cluster resetting. I've updated REDME.md and changed the liveness probe command which needs to run with root user credentials. Please review

BLasan committed:
Fix MySQL Access Denied Issue (FINERACT-1294)

vorburger commented on kubernetes/fineractmysql-deployment.yml:
```diff
@@ -96,11 +96,7 @@ spec:
-                - ping
-                - -h
-                - localhost
+              command: ["sh","-c","mysqladmin ping -h localhost -uroot -p${MYSQL_ROOT_PASSWORD}"]
```
@BLasan sorry I'm not getting something here- where/by what is this MYSQL_ROOT_PASSWORD defined? Tx. /Cc @xurror

vorburger replied:
@BLasan oh wait I am dumb, this `MYSQL_ROOT_PASSWORD` **IS** set by us, in https://github.com/apache/fineract/blob/develop/kubernetes/fineractmysql-deployment.yml#L92, now I get how this works! Then this change makes perfect sense - LGTM and I'm mergint it now. FYI @xurror

BLasan commented:
@vorburger Sir could you please take a look?
